task/build-helm-chart-oci-ta/0.1/build-helm-chart-oci-ta.yaml

```diff
@@ -0,0 +1,137 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-helm-chart-oci-ta
+spec:
+  description: Package and push a Helm chart to an OCI repository
```

Comment:
The description on the OCI TA variant is different from the non-OCI TA task. Did you run the `generate-ta-tasks.sh` script to generate this?

Reply:
I now made it so that everything is the same (description and readme for both tasks).